resources/metadata/influx.yaml

```diff
@@ -0,0 +1,66 @@
+metadata:
+  name: influxWriteData
+  description: Writes metrics to influxdb
+  longDescription: |
+    In this step, the metrics are written to the timeseries database [InfluxDB](https://www.influxdata.com/time-series-platform/influxdb/)
+spec:
+  inputs:
+    secrets:
+      - name: influxTokenId
+        description: Influxdb token for authentication to the InfluxDB. In 1.8 version use 'username:password' instead.
+        type: jenkins
+    params:
+      - name: serverUrl
+        type: string
+        description: Base URL to the InfluxDB server
+        scope:
+          - PARAMETERS
+          - STAGES
+          - STEPS
+          - GENERAL
+        mandatory: true
+      - name: authToken
+        type: string
+        description: Token to authenticate to the Influxdb
+        scope:
+          - PARAMETERS
+          - STAGES
+          - STEPS
+        mandatory: true
+        secret: true
+        resourceRef:
```

Comment:
let's also add a Vault reference here so that the credentials can be retrieved from Vault as well

Reply:
done